include/boost/fusion/container/generation/ignore.hpp

```diff
@@ -27,7 +29,7 @@ namespace boost { namespace fusion
     }
 
     //  "ignore" allows tuple positions to be ignored when using "tie".
-    BOOST_CONSTEXPR detail::swallow_assign const ignore = detail::swallow_assign();
+    BOOST_CONSTEXPR_OR_CONST detail::swallow_assign ignore = detail::swallow_assign();
 }}
```

Comment:
Here is: [Macros that allow use of C++11 features with C++03 compilers](http://www.boost.org/doc/libs/1_58_0/libs/config/doc/html/boost_config/boost_macro_reference.html#boost_config.boost_macro_reference.macros_that_allow_use_of_c__11_features_with_c__03_compilers).